x/datadeal/abci.go

```diff
@@ -40,7 +40,7 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 				panic("invalid verifiable CID consensus value")
 			}
 
-			keeper.AddDataDeliveryQueue(ctx, dataSale.VerifiableCid, dataSale.DealId, oracleKeeper.GetVotingPeriod(ctx).VotingEndTime)
+			keeper.AddDataDeliveryQueue(ctx, dataSale.VerifiableCid, dataSale.DealId, dataSale.DeliveryVotingPeriod.VotingEndTime)
```

Comment:
I think `DeliveryVotingPeriod` set a here.
```go
dataSale.DeliveryVotingPeriod = oracleKeeper.GetVotingPeriod(ctx)
keeper.AddDataDeliveryQueue(ctx, dataSale.VerifiableCid, dataSale.DealId, dataSale.DeliveryVotingPeriod.VotingEndTime)
```

Reply:
Good. I was just writing this comments as well. 👍

Reply:
Ah, I missed it. I'll add it! Thanks!

Reply:
Fixed in [73769b8](https://github.com/medibloc/panacea-core/pull/456/commits/73769b88a9b7749a80f28c42c7f28e7ee838d8a9).